### ISSUE TITLE
Alert - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036569777)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7780]
### Violation: 
 Ensure parts of graphical objects essential for understanding content have sufficient contrast 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/0388d94a-6bbb-4622-9210-52b6346940c0)

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 5 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Alert - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036569777) 
- Module: Alert 
**Issue**
The "Yellow" icon("medium duration" alert) with a color contrast ratio below 3.00:1

Foreground: #EFD833
Background: #FFFFFF
Contrast ratio: 1.4:1

**User Impact**
Users with low vision will have difficulty identifying this content.

**Code Reference** 
 ```
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg">
...
</svg>

[CSS]
host([color="yellow"]) .container .alert-icon {
    color: var(--calcite-ui-warning);
}


 ``` 
**Suggestion**
Ensure parts of graphical objects essential for understanding content have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying objects include lines in a chart, meaningful icons, and annotations within an image.

Graphics that require particular presentation to preserve their meaning are exempt from this requirement.

To evaluate color contrast, see the Accessible Color Picker extension: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by macandcheese** [comment]
Thoughts on closing this? - We've always said that these icons are not meaningful icons, and instead supportive in nature and by themselves do not convey the meaning of `kind` of Alert - the component does not have an `icon` displayed by default, it is opt-in (either the default icons or a custom one).

The meaning and severity of an Alert's message should be conveyed by the content slotted in title / subtitle slot.

If we do decide these are meaningful, the issue is really with the color values used for our `kind` properties.

**Comment by geospatialem** [comment]
This is handled with an `aria-hidden="true"` as part of the svg, so it is no longer valid.
